tests/regression/test_expressions.py

```diff
@@ -523,7 +524,6 @@ def test_math_functions(expr, value):
     actual = Function(V)
 
     actual.assign(eval(expr))
-    from math import *          # noqa: F401
```

Comment:
I think this doesn't DTRT?  Because those math functions are shadowed by the ones imported from ufl (via the `from firedrake import *`).  So maybe there's no need for `from math import *`, because UFL constant folds anyway.